### PR TITLE
Add remove done items

### PR DESCRIPTION
**Pull request by liaochuan**
Added a button to clean up the to-dos that have been completed in the to-do list, enable it through the configuration, and remove the completed items after clicking.
image
image
image

### CODE REVIEW

**Comment by RoccoSmit** [comment]
@liaochuan, can you please fix the static checks that are failing.
On your local you can run `pnpm run lint`

**Comment by liaochuan** [comment]
> @liaochuan, can you please fix the static checks that are failing. On your local you can run `pnpm run lint`

It's been dealt with

File: web/src/components/MemoActionMenu.tsx [hunk 1]
```diff
@@ -113,6 +116,42 @@ const MemoActionMenu = (props: Props) => {
     }
   };
 
+  const handleRemoveCompletedTaskListItemsClick = async () => {
+    const confirmed = window.confirm(t("memo.remove-completed-task-list-items-confirm"));
+    if (confirmed) {
+      const newNodes = JSON.parse(JSON.stringify(memo.nodes));
+      for (let i = 0; i < newNodes.length; i++) {
```
Comment (boojack) [inline web/src/components/MemoActionMenu.tsx:123]: it's better to use `for...of`

**Review by boojack — APPROVED** [review]
LGTM, thanks for your contribution!

File: web/src/components/MemoActionMenu.tsx (hunk 1 above)
Comment (boojack) [reply web/src/components/MemoActionMenu.tsx:123]: I'll update in a seperate commit.

**Pull request merged**